lms/templates/instructor/instructor_dashboard_2/student_admin.html

```diff
@@ -39,49 +36,60 @@ <h3 class="hd hd-3">${_("Student-specific grade inspection")}</h3>
 <div class="student-grade-container action-type-container">
   <h3 class="hd hd-3">${_("Student-specific grade adjustment")}</h3>
   <div class="request-response-error"></div>
-  <p>
-  <label>
+  <label for="student-select-grade">
   ${_("Specify the {platform_name} email address or username of a student here:").format(platform_name=settings.PLATFORM_NAME)}
+      </br>
   <input type="text" name="student-select-grade" placeholder="${_("Student Email or Username")}">
   </label>
-  </p>
   <br>
+  </br>
 
-  <label> ${_("Specify a problem in the course here with its complete location:")}
+  <label for="problem-select-single"> ${_("Specify a problem in the course here with its complete location:")}
+      </br>
   <input type="text" name="problem-select-single" placeholder="${_("Problem location")}">
   </label>
+    </br>
+    </br>
 
   ## Translators: A location (string of text) follows this sentence.
-  <p>${_("You must provide the complete location of the problem. In the Staff Debug viewer, the location looks like this:")}<br/>
-  <code>block-v1:edX+DemoX+2015+type@problem+block@618c5933b8b544e4a4cc103d3e508378</code></p>
+  <label for="problem-location-example-1">${_("You must provide the complete location of the problem. In the Staff Debug viewer, the location looks like this:")}</label></br>
+  <code name="problem-location-example-1">block-v1:edX+DemoX+2015+type@problem+block@618c5933b8b544e4a4cc103d3e508378</code>
 
-  <p>
+    </br>
+    </br>
+  <label for="single-user-actions">
   ${_("Next, select an action to perform for the given user and problem:")}
-  </p>
+  </label>
 
-  <p>
+    </br>
+    <span name="single-user-actions">
   <input type="button" name="reset-attempts-single" value="${_("Reset Student Attempts")}" data-endpoint="${ section_data['reset_student_attempts_url'] }">
 
   %if settings.FEATURES.get('ENABLE_INSTRUCTOR_BACKGROUND_TASKS') and section_data['access']['instructor']:
     <input type="button" name="rescore-problem-single" value="${_("Rescore Student Submission")}" data-endpoint="${ section_data['rescore_problem_url'] }">
+    <input type="button" name="rescore-problem-if-higher-single" value="${_("Rescore Student Submission If Higher")}" data-endpoint="${ section_data['rescore_problem_url'] }">
   %endif
-  </p>
+</span>
 
-  <p>
+    </br>
+    </br>
   %if section_data['access']['instructor']:
-    <label> ${_('You may also delete the entire state of a student for the specified problem:')}
-    <input type="button" class="molly-guard" name="delete-state-single" value="${_("Delete Student State for Problem")}" data-endpoint="${ section_data['reset_student_attempts_url'] }"></label>
+    <label for="delete-state-single"> ${_('You may also delete the entire state of a student for the specified problem:')}
```

Comment:
"may" -> "can", and reword:
"You can also delete a student's state for the specified problem".